docs/groovy/how-to-guides/install-and-use-java-packages.md

```diff
@@ -2,133 +2,123 @@
+The examples in this guide add the [Plexus Common Utilities](https://codehaus-plexus.github.io/plexus-utils/) library to a Deephaven Community instance.
 
-In this section, we add the [Java CryptoCompare API client](https://mvnrepository.com/artifact/com.github.jeffreytai/cryptocompare-api-wrapper) to a custom Dockerfile and reference it from Deephaven so that we can use it in more than one session.
+## Docker
 
-In order to use packages more than once, you can create a custom Docker image, and then use that image in Deephaven. The steps for accomplishing this differ slightly depending on how you launch Deephaven. Let's start with the steps that are common between both.
+If you [Run Deephaven with Docker](../tutorials/docker-install.md), you can either build a custom Docker image or mount a volume containing the JAR into the container.
 
-### Prerequisites
+### Build a custom Docker image
 
-Before a custom Docker image can be built, you must acquire the necessary base images. This process differs based upon how you launch Deephaven:
+To build a custom Docker image, create a Dockerfile that downloads the JAR and mounts it into the container. The following `Dockerfile` mounts `plexus-utils-4.0.2.jar` into `/apps/libs` in the container:
 
-- If you [launch from pre-built images](../tutorials/docker-install.md), ensure you have run the following command to download the necessary base images:
-
-```shell
-docker compose pull
+```dockerfile
+FROM ghcr.io/deephaven/server-slim:latest
+RUN curl --output /apps/libs/plexus-utils-4.0.2.jar https://repo1.maven.org/maven2/org/codehaus/plexus/plexus-utils/4.0.2/plexus-utils-4.0.2.jar
 ```
 
-- If you [launch from source code](./launch-build.md), ensure you have built the project so that you have the necessary base images.
-
-### Create a custom Dockerfile
+You then need to build your image. If you use Docker without Compose, run `docker build` and `docker run`:
 
-To begin, create a new directory. This directory should not be in a Deephaven deployment directory. You can name it whatever you'd like. For this guide, we'll name ours `deephaven-custom`.
-
-```shell
-mkdir deephaven-custom
-cd deephaven-custom
+```bash
+docker build --tag deephaven-plexus-utils .
+docker run --rm --name deephaven-with-plexus-utils -p 10000:10000 deephaven-plexus-utils
 ```
 
-Now, in this directory, create a file called `Dockerfile`. `Dockerfile` should use `ghcr.io/deephaven/server` as the base image and should contain a recipe for installing the new package. When adding the [Java CryptoCompare API client](https://mvnrepository.com/artifact/com.github.jeffreytai/cryptocompare-api-wrapper), it looks like this:
+Alternatively, you can build your image manually and reference it in a Docker Compose YAML file:
```

Comment:
I don't think you should have these two docker compose examples; showing how to do it with plain `docker` should be "good enough" IMO.